README.md

```diff
@@ -13,6 +13,7 @@ This repo can be installed on either the same VM as the main LORIS codebase, or
 On <u>Ubuntu</u>, DICOM toolkit will be installed by the imaging install script (step 4 below). This script will _apt-get install dcmtk_.   
```

Comment:
Is there no need to install Minc toolkit before the imaging install script runs?